app/controllers/favorites_controller.rb

```diff
@@ -1,6 +1,10 @@
 class FavoritesController < ApplicationController
   include ActionView::Helpers::TextHelper
 
+  def index
+    favorites
+  end
+
   def update
     pet = Pet.find(params[:pet_id])
     # @favorite = Favorite.new(session[:favorite])
```

Comment:
looks like we can remove this now. what do you think?

Reply:
Absolutely; all comments are now removed; please confirm on your next code-review

Reply:
Confirmed!